CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+## Welcome!
+
+We're excited that you're thinking about contributing to Sprintly-UI.
+In order to make contributing as friction-free as possible, please read through and consider the following guidelines.
+
+#### Roadmap and issues
+Please refer to [the roadmap](roadmap.md) before starting work on new components that you plan for inclusion in this repo. This will help us avoid duplicate efforts. If there's something that we're already planning via the roadmap that you would nevertheless like to work on, let us know and we'll see if we can make that work with our dev schedule.
```

Comment:
Seems like this should encourage people to collaborate with us, rather than "We'll fit you into our schedule."